legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/preeval.pure

```diff
@@ -187,7 +187,7 @@ function <<access.private>> meta::pure::router::preeval::prevalInternal(item : A
                                                             a : meta::pure::functions::collection::AggregateValue<Any,Any,Any>[*]| ^InstanceValue(multiplicity = $ve.multiplicity, genericType = $ve.genericType, values = $varValue)->evaluateAndDeactivate(),
                                                             a : meta::pure::tds::BasicColumnSpecification<Any>[*]| ^InstanceValue(multiplicity = $ve.multiplicity, genericType = $ve.genericType, values = $varValue)->evaluateAndDeactivate(),
                                                             r : meta::pure::graphFetch::RootGraphFetchTree<Any>[1]| ^InstanceValue(multiplicity = $ve.multiplicity, genericType = $ve.genericType, values = $varValue)->evaluateAndDeactivate();,
-                                                            b : meta::external::shared::format::binding::Binding[1]| ^InstanceValue(multiplicity = $ve.multiplicity, genericType = $ve.genericType, values = $varValue)->evaluateAndDeactivate();,
+                                                            b : meta::pure::store::Store[1]| ^InstanceValue(multiplicity = $ve.multiplicity, genericType = $ve.genericType, values = $varValue)->evaluateAndDeactivate();,

```

Comment:
Binding is meant to be removed from Store hierarchy so please add a new match condition instead of replacing. Thanks

Reply:
As discusses, I will submit a PR later today / tomorrow morning re-adding Binding.